packages/html-spec/src/forms/forms.json

```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "component": "forms",
+        "selector": [
+            ".k-form"
+        ],
+        "states": [
+            {
+                "name": "error",
+                "selector": [
+                    ".k-form-error"
+                ]
+            }
+        ],
+        "options": [
```

Comment:
The form is vertical by default, however, the `k-form-horizontal` class can be used change the orientation. Please describe it here.